Makefile

```diff
@@ -696,6 +705,7 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 		--app 'Mobile application' apps/mobile/index.html 'The mobile example application for GeoMapFish.' \
 		--app 'Desktop application' apps/desktop/index.html 'The desktop example application for GeoMapFish.' \
 		--app 'Alternate desktop application' apps/desktop_alt/index.html 'An alternate desktop example application for GeoMapFish.' \
+		--app 'Alternate mobile application' apps/mobile_alt/index.html 'An alternate mobile example application for GeoMapFish.' \
```

Comment:
I see that you already created a PR for it.. so I will close my PR and remove the branch)